otj-server-core/src/main/java/com/opentable/server/EmbeddedJettyBase.java

```diff
@@ -263,6 +267,12 @@ private ConnectorInfo createConnector(Server server, String name, IntSupplier po
         final HttpConnectionFactory http = new HttpConnectionFactory(httpConfig);
 
         if (ssl != null) {
+            if (!CollectionUtils.isEmpty(excludedProtocols)) {
+                LOG.warn("Excluding following protocols:");
```

Comment:
Probably should log at the info level here too

Reply:
Warn is higher than info ain't it

Reply:
sticking with warn, same level of logging is used in otj server to note the fact its' running with old ciphers.